packages/back-end/src/api/openapi/schemas/SavedGroup.yaml

```diff
@@ -0,0 +1,27 @@
+type: object
+required:
+  - id
+  - dateCreated
+  - dateUpdated
+  - groupName
+  - attributeKey
+  - values
+properties:
+  id:
+    type: string
+  dateCreated:
+    type: string
+    format: date-time
+  dateUpdated:
+    type: string
+    format: date-time
+  groupName:
```

Comment:
This is called `name` in the create and update endpoints.  We should keep it consistent here.